css/dashboards/dashboards.css

```diff
@@ -508,11 +508,10 @@
     border: none;
     color: var(--highcharts-neutral-color-100);
     cursor: pointer;
-    justify-content: space-between;
+    justify-content: space-around;
     display: flex;
-    min-width: 100%;
-    column-gap: 8px;
-    white-space: nowrap;
+    min-width: 100px;
+    gap: 8px;
```

Comment:
This is the same change you initially had on the PR to fix the cypress tests, so just make the `gap: 8px`, does not end up in master